src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.MultiAgent.cs

```diff
@@ -565,7 +565,7 @@ private static size_t CurlReceiveHeadersCallback(IntPtr buffer, size_t size, siz
                             if (colonIndex > 0)
                             {
                                 string headerName = responseHeader.Substring(0, colonIndex);
-                                string headerValue = responseHeader.Substring(colonIndex + 1);
+                                string headerValue = responseHeader.Substring(colonIndex + 1).Trim();
```

Comment:
I'm following what's being done in WinHttpHandler:
https://github.com/dotnet/corefx/blob/master/src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpResponseParser.cs#L201
@davidsh, is WinHttpHandler too aggressive here, or have I misunderstood?